virtualcluster/pkg/controller/controllers/metrics.go

```diff
@@ -0,0 +1,30 @@
+package controllers
```

Comment:
Please add file headers for all newly created files.

Reply:
Specifically the license contents @m-messiah , thanks!

Reply:
will be added in https://github.com/kubernetes-sigs/cluster-api-provider-nested/pull/292 (with linter)